Python/sysmodule.c

```diff
@@ -2123,12 +2123,20 @@ extern PyObject *_Py_GetObjects(PyObject *, PyObject *);
 sys._clear_type_cache
 
 Clear the internal type lookup cache.
+
+This function is deprecated and will be removed in a future python version.
```

Comment:
I don't think we need this comment (make sure to also rerun argument clinic when removing it :))
```suggestion
```